src/modules/esl-event-listener/core/targets/inert.target.event.ts

```diff
@@ -0,0 +1,43 @@
+import {overrideEvent} from '../../../esl-utils/dom/events/misc';
+
+/**
+ * Event names that could be triggered by {@link ESLWheelTarget}
+ */
+export type WheelEventName = 'longwheel';
+
+/**
+ * Describes scroll information provided with {@link ESLWheelTarget}
+ */
+export interface ESLWheelEventInfo {
+  /** Distance scrolled along the x-axis */
+  deltaX: number;
+  /** Distance scrolled along the y-axis */
+  deltaY: number;
+  /** First wheel event */
+  startEvent: WheelEvent;
+  /** Last wheel event */
+  endEvent: WheelEvent;
```

Comment:
I don't think we interested in original events at least in first and last event specifically.
We can consider a list of all aggregated events, or skip this option for now
CC: @abarmina to make API notes in story ticket